packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -59,6 +60,7 @@ const sequelize = initDBPool(Config.getDBCreds());
  */
 router.post(
   "/directory/rebuild",
+  requestLogger,
```

Comment:
This route is hit with the scheduled lambda once a day. Should we even log this?